selftests/functional/test_plugin_jobscripts.py

```diff
@@ -93,14 +106,37 @@ def test_status_non_zero(self):
         config = script.TemporaryScript("non_zero.conf",
                                         SCRIPT_NON_ZERO_CFG % self.pre_dir)
         with config:
-            cmd = './scripts/avocado --config %s run passtest.py' % config
+            cmd = './scripts/avocado --config %s run /bin/true' % config
```

Comment:
There should be no reason for using `/bin/true` here.  We try to keep the functional tests contained to the example tests we ship.  In fact I tried with `passtest.py` and it works for me.  If you're experiencing errors, it may be due to the lack of `make develop`.